Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitNonce.swift

```diff
@@ -24,8 +24,8 @@ import BraintreeCore
         guard let nonce = json["nonce"].asString() else { return nil }
         
         self.nonce = nonce
-        self.ibanLastFour = json["details"]["last4"].asString()
-        self.customerID = json["details"]["customerId"].asString()
+        self.ibanLastFour = json["details"]["ibanLastChars"].asString()
+        self.customerID = json["details"]["merchantOrPartnerCustomerId"].asString()
```

Comment:
Should we add this on Android too?

Reply:
It's correct on Android already, just got missed when the name changed a few times on iOS 😢 : https://github.com/braintree/braintree_android/blob/master/SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitNonce.java#L20

Reply:
Nice aight yeah the PRs are exactly the same for the most part wanted to make sure 👍